CHANGELOG.md

```diff
@@ -4,6 +4,42 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.3.0 - 2018-10-22
+
+### Added
+
+- Asset state is loaded from the install directory, allowing for a [staged
+  install](docs/user/overview.md#multiple-invocations).
+- The installer pushes a `bootstrap-complete` event to the
+  `kube-system` namespace when the boostrapping process completes and
```

Comment:
There is a bug with this. The bootstrap reports completed long before it actually completes :(